contracts/contracts/GrantRound.sol

```diff
@@ -52,6 +55,7 @@ contract GrantRound {
    * @param _payoutAdmin Grant round administrator that has permission to payout the matching pool
    * @param _registry Address that contains the grant metadata
    * @param _donationToken Address of the ERC20 token in which donations are made
+   * @param _donationToken Address of the ERC20 token for accepting matching pool contributions
```

Comment:
Typo. Also would update the `_donationToken` and `_matchingToken` comments to match any changes made to their state variable comments above

```suggestion
   * @param _matchingToken Address of the ERC20 token for accepting matching pool contributions
```